assets/aml-benchmark/components/batch-inference-preparer/spec.yaml

```diff
@@ -47,14 +51,19 @@ outputs:
   formatted_data:
     type: mltable
     description: Path to the folder where the payload will be stored.
+  ground_truth_metadata:
+    type: uri_folder
+    description: Path to the folder where the payload will be stored.
```

Comment:
This has the same description as `formatted_data`- both says path to the folder where the payload will be stored. I am assuming this description has to be updated.

Reply:
changed